.circleci/config-approval.yml

```diff
@@ -274,15 +274,15 @@ workflows:
           requires:
             - build_test
       - push_image:
-          name: push_image_labs_preview
```

Comment:
I named push_image_labs_preview because Labs and Preview shares the same ACR. When you want to push some docker image for Labs and Preview environment they are stored in the same repository. Does it make sense to apply this rename?